my-first-react-app/src/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './app.js';
+import Hello from './hello.js';
```

Comment:
This is all good, but we don't actually need to bring in `hello.js` here, we only need to import it where we use it.

Also, regarding your file names, you have 

```import App from './app.js';```

Though, the file name is `App.js`, not `app.js` on a `unix` operating system, this is ok, but on Linux, this would cause errors, it is best to pick one naming convention and stick with it, all capitalized , or all lowercase for instance.

Since create-react-app favors capitalized file names, I would suggest that.